ReactAndroid/src/androidTest/java/com/facebook/react/testing/ReactAppTestActivity.java

```diff
@@ -34,18 +32,15 @@
 import com.facebook.react.shell.MainReactPackage;
 import com.facebook.react.testing.idledetection.ReactBridgeIdleSignaler;
 import com.facebook.react.testing.idledetection.ReactIdleDetectionUtil;
-import com.facebook.react.uimanager.UIManagerModule;
-import com.facebook.react.uimanager.ViewManager;
 import com.facebook.react.uimanager.ViewManagerRegistry;
-import com.facebook.react.uimanager.events.EventDispatcher;
 import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 import javax.annotation.Nullable;
 
 
-public class ReactAppTestActivity extends FragmentActivity
+public class ReactAppTestActivity extends ReactActivity
     implements DefaultHardwareBackBtnHandler, PermissionAwareActivity {
```

Comment:
```
implements DefaultHardwareBackBtnHandler, PermissionAwareActivity
```

shouldnt be needed since `ReactActivity` implements them already:

```
public abstract class ReactActivity extends FragmentActivity
    implements DefaultHardwareBackBtnHandler, PermissionAwareActivity {
```

`invokeDefaultOnBackPressed` method can also be removed as the one here will be the same as in the parent class.

Reply:
@hey99xx thanks, removed